client/contracts/truffle.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  networks: {
+    ganache: {
+      host: '127.0.0.1',
```

Comment:
Ganache is a testing sandbox, if I remember correctly. Can this be used for integration tests?

Reply:
Yes, it can be used for integration tests because it simulates ethereum network. Ganache replaced testrpc in truffle 4.0 if I remember correctly